exporter/coralogixexporter/testdata/config.yaml

```diff
@@ -31,3 +30,14 @@ coralogix/all:
   application_name: "APP_NAME"
   subsystem_name: "SUBSYSTEM_NAME"
   timeout: 5s
+
+coralogix/domain:
```

Comment:
If setting both the signal specific endpoints and domain is intended can you add a test case for that scenario